firmware/lib/Axis/src/Axis.h

```diff
@@ -0,0 +1,63 @@
+#ifndef AXIS_H
+#define AXIS_H
+
+#include "Debug.h"
+#include "Kinematics.h"
+// -------------------------------- X AXIS SETUP -------------------------------- //
+#define ACCEL_X                 100
+#define VEL_MIN_X               100
+#define VEL_MAX_X               1000
+#define STEP_PIN_X              5 // need to know the register for fast operation pin 5 - PC25 
+#define DIR_PIN_X               6 // need to know the register for fast operation pin 6 - PC24
+#define ENCODER_PIN_A_X         7
+#define ENCODER_PIN_B_X         8
+#define LIMIT_SW_HOME_PIN_X     9
+#define LIMIT_SW_FAR_PIN_X      10
```

Comment:
It seems like most if not all of these defines are only used in the Axis.cpp file. If that's the case they don't need to be defined in the header and can be moved to the cpp file